components/webdriver_server/lib.rs

```diff
@@ -100,14 +106,14 @@ struct WebDriverSession {
 
     /// Time to wait for injected scripts to run before interrupting them.  A [`None`] value
     /// specifies that the script should run indefinitely.
-    script_timeout: Option<u32>,
+    script_timeout: Option<u64>,
 
     /// Time to wait for a page to finish loading upon navigation.
-    load_timeout: u32,
+    load_timeout: Option<u64>,
```

Comment:
i changed it to an option to be consistent with the TimeoutParameters in webdriver